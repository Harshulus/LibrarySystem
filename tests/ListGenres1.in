list genres
add book SampleCollection.csv 111111
list genres
genre
genre comedy
