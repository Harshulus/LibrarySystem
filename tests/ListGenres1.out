user: No books in system.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Historical Fiction

user: Provide genre type.

user: No books in system.

